app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,8 @@
+<li>
+    <p><%= post.title %></p>
+    <p><%= post.content %></p>
+    <h6>Updated at: <%= post.updated_at %></h6>
+    <% if @logged_in %>
```

Comment:
- [ ] There shoud be absolutely NO logic in views.


![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)


View-related logic can be encapsulated in View helpers (app/helpers/*_helper.rb)
Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information.